InvenTree/templates/js/translated/label.js

```diff
@@ -92,6 +94,16 @@ function selectLabel(labels, items, options={}) {
         },
         function(item) {
             return item.pk;
+        },
+        null,
+        function(item) {
+            if (options.key == 'part')
+                return item.pk == "{{ default_part_template_pk }}";
```

Comment:
You can access user settings directly, this might be more readable as:

```javascript
return item.pk == user_settings.DEFAULT_LOCATION_LABEL_TEMPLATE;
```